python/python/tests/test_async_client.py

```diff
@@ -2574,6 +2574,28 @@ async def test_pfadd(self, redis_client: TRedisClient):
             await redis_client.pfadd("foo", [])
 
 
+class TestMultiKeyCommandCrossSlot:
+    @pytest.mark.parametrize("cluster_mode", [True])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_multi_key_command_returns_cross_slot_error(
+        self, redis_client: RedisClusterClient
+    ):
+        # TODO brpop, bz*, zdiff, sdiff and others - all rest multi-key commands except ones tested below
```

Comment:
There are some existing tests that are testing for the crossslot error, can you move them here? eg blpop/brpop, zdiffstore etc

Reply:
Added, thanks